scripts/flag_reviews.py

```diff
@@ -13,7 +13,7 @@ def main():
     SPREADSHEET_RANGE ='A1:Z3000'
     credentials_file_name = 'credentials.json'
     locality_by_place_id = {}
-    DATABASE_URL = ''
+    DATABASE_URL = 'postgres://u4s5nl6jojreas:p3b365a9a7cc291943314402f2a06d27acba79d932513d7400541f51c8a754fe9@ec2-54-210-160-194.compute-1.amazonaws.com:5432/d8a2u67pgoohhq'
```

Comment:
let's not commit this please, i don't want to risk a ddos attack. let's use environment variables or something like that instead. we will need to roll our database credentials now since they are committed to git history

Reply:
just rotated the credentials. these will be invalid

Reply:
Good catch, sorry I missed these!